tools/ovc/openvino/tools/ovc/moc_frontend/pipeline.py

```diff
@@ -90,156 +92,205 @@ def check_places_are_same(places_original: List[Place], places_new: List[Place])
-            except NotImplementedFailure as e:
-                # some frontends might not implement this method
-                log.warning('Could not add an additional name to a tensor pointed to by \'{}\'. Details: {}'.format(
-                    new_input['input_name'], str(e)))
+    if getattr(argv, "framework", None) == "pytorch":
+        iplaces = []
+        for idx, input_info in enumerate(argv.input):
+            if getattr(input_info, "name", None):
+                place = input_model.get_place_by_tensor_name(input_info.name)
+                if not input_info.shape and not input_info.type:
+                    # If we received place by name, we need to use it for FE to verify
+                    # that such name exist, otherwise we silently ignore it.
+                    # Using dynamic shape should be safe, because FE will not overwrite
+                    # the shape that was produced after conversion, but merge it, so
+                    # dynamic shape will not change anything.
+                    input_model.set_partial_shape(place, PartialShape.dynamic())
+            else:
+                place = input_model.get_place_by_input_index(idx)
+            iplaces.append(place)
+            if input_info.shape is not None:
+                input_model.set_partial_shape(place, input_info.shape)
+            if input_info.type is not None:
+                input_model.set_element_type(place, input_info.type)
+        model_inputs = input_model.get_inputs()
+        def merge_inputs(inputs, to_set_list):
+            # use input places instead of obtained by index if they are the same
+            res = []
+            for p in to_set_list:
+                found = False
+                for i in inputs:
+                    if p.is_equal(i):
+                        res.append(i)
+                        found = True
+                        break
+                if not found:
+                    res.append(p)
+            return res
+        iplaces = merge_inputs(model_inputs, iplaces)
+        # Currently this only work to reorder inputs/outputs
+        to_override_all_inputs = check_places_are_same(model_inputs, [{"node": p} for p in iplaces])
+        to_override_all_outputs = False
+        if argv.output:
+            oplaces = []
+            _outputs = fe_output_user_data_repack(input_model, argv.output, moc_front_end.get_name())
+            for out_desc in _outputs:
+                oplaces.append(out_desc["name"])
+            model_outputs = input_model.get_outputs()
+            to_override_all_outputs = check_places_are_same(model_outputs, [{"node": p} for p in oplaces])
+        if to_override_all_inputs and to_override_all_outputs:
+            input_model.extract_subgraph(iplaces, oplaces)
+        elif to_override_all_inputs:
+            input_model.override_all_inputs(iplaces)
+        elif to_override_all_outputs:
+            input_model.override_all_outputs(oplaces)
+    else:
```

Comment:
let us do not create `else` branch just call ` moc_front_end.convert(input_model)` in if.
This way you localize PyTorch code.

Reply:
done